mantidproject/mantid
# Remove last modified dates from algorithm documentation pages

**Description of work.**

The source link dates were parsed from the git log of the source
files and would include *any* change to the file regardless
of whether the change was meaningful, e.g. a formatting
change.
The preferred option going forward is for each page to
document any change in behaviour across versions in its
own changelog on the page.



**To test:**

* Build the docs and check an algorithm page, e.g. Rebin and it should no longer have the  date is was modified.
* I would encourage running cmake with the following options to get the quickest docs build:
  * `DOCS_PLOTDIRECTIVE=OFF`
  * `DOCS_SCREENSHOTS=OFF`
  * `DOCS_DOTDIAGRAMS=OFF`
  * `DOCS_MATH_EXT=sphinx.ext.mathjax`

*There is no associated issue.*



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
